markdown/articles/2021/08/22/Non-blue-fair-decks.md

```diff
@@ -0,0 +1,429 @@
+
+#### Post Doomsday 
+
+Passing the turn piles can be a tad risky, because Elves can be really explosive. Especially Turn 3 or later.  
+In the past some Elves lists played Endurance or Crop Rotation for Ipnu Rivulet, but as long as they stay absent from Sideboards, you only have to play around Collector Ouphe.
+
+## Aluren
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Endurance}} {{!Force of Will}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Force of Negation}} {{!Opposition Agent}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Aluren is a pretty slow combo deck. To go off they need atleast 4 Mana, Aluren and two of their combo creatures or Acererak the Archlich. This should give you enough time to find Doomsday and set up a safe kill.  
+Resolving Doomsday should not be too hard, Aluren plays 4 Force of Will main and 2 Force of Negation in the sideboard. That's often not enough when they also try to set up their own kill. In game 1 one counterspell is often enought to force your Doomsday through. In game 2 you want enough interaction to beat atleast 2 Force effects from their side and try to keep an eye out for Opposition Agent.
+
+#### Post Doomsday 
+
+Many lists play Endurance, like most green decks in the current meta, so prepare to play around atleast one copy.  
+If they fought about your Doomsday, they often don't have enough resources to go off, but that is not always the case, especially since Acererak the Archlich joined the format. Should they still be able to go off and you have enough cyclers in hand to cycle to Thassa's Oracle, be careful with casting it of Aluren. You have to cast it when their combo creature is on the stack, else they can react and combo off while the Thassa's Oracle trigger is still no the stack.
+
+## Reanimator
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Grief}} {{!Chancellor of the Annex}} {{!Cabal Therapy}} {{!Thoughtseize}} {{!Unmask}}</row>
+
+#### Sideboard
+
+<row variant="hand"> {{!Iona, Shield of Emeria}}</row>
+
+### Gameplay
+
+#### Pre Doomsday
+
+Do NOT keep hands without countermagic. You'll need it to protect your own combo and fighting off theirs. With all their discard effects one free counter is often not enough. The only reason to keep hands without protection are turn 1 kills.  
+When fighting their combo, try always to counter the Entomb effects, without a good target their reanimation spells are useless. And as a little sidenote, most lists play only 8 ways to creatures into the graveyard, but 12 reanimation spells.
+
+#### Post Doomsday
+
+One of the scariest creatures they can reanimate post Doomsday is Ashen Rider, so try to be carefull with your lands and don't be stingy with your lotus petals.
+
+## Sneak and Show
+
+### Gameplan
+
+### Cards to keep in mind
+
+#### Maindeck
+
+<row variant="hand"> {{!Daze}} {{!Force of Will}}</row>
```

Comment:
Add Sneak attack. The possibility of a hasty dude can sometimes be the kicker,

Reply:
added